.github/scripts/update_release_branch.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+set -e
+
+release_branch=$1
+target_req_file=".github/workflows/nightly-release.yml"
```

Comment:
This doesn't exist right now because we can't run the release for spark while it still uses circleci.  Release related updates need to happen here in relation to using GHA for integration tests.  I would do that separately though.

Reply:
I agree. I think we should focus on just the integration tests for this PR.